### ISSUE TITLE
Trying to drag a token (as a player) while the game is paused causes a ping

### ISSUE THREAD

**Issue by ThiefMaster** [foundryvtt/foundryvtt#10456]
### What happened?

When you try to drag your token while the game is paused, it triggers a ping (I guess the mouse movement is ignored, so it's considered a long click at the place where you initially clicked).

### What ways of accessing Foundry can you encounter this issue in?

- [ ] Native App (Electron)
- [ ] Chrome
- [X] Firefox
- [ ] Safari
- [ ] Other

### Reproduction Steps

- Pause the game (as GM)
- Try to move a token you can usually control (as player)

### What core version are you reporting this for?

11 build 315

### Bug Checklist

- [ ] The issue occurs while all Modules are disabled

**Comment by aaclayton** [comment]
Yeah, this is a case where the interaction framework is "working as intended" - but the outcome is perhaps not intuitive. What is your opinion about what *should* happen instead?

**Comment by ThiefMaster** [comment]
Either starting a selection like anywhere else or not doing anything would be fine IMHO
